File: fitmily_backend/user-service/src/main/java/com/d208/fitmily/domain/user/dto/request/JoinRequest.java
package com.d208.fitmily.domain.user.dto.request;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class JoinRequest {

    private String loginId;
    private String password;
    private String nickname;
    private String birth;
    private Integer gender;

}

File: fitmily_android/app/src/main/java/com/ssafy/fitmily_android/model/dto/request/auth/JoinRequest.kt
package com.ssafy.fitmily_android.model.dto.request.auth

import com.google.gson.annotations.SerializedName

data class JoinRequest(
    @SerializedName("loginId") val userLoginId: String
    , @SerializedName("password") val userPw: String
    , @SerializedName("nickname") val userNickname: String
    , @SerializedName("birth") val userBirth: String
    , @SerializedName("gender") val userGender: Int
)